utils/build-script-impl

```diff
@@ -542,10 +542,10 @@ function set_build_options_for_host() {
             esac
 
             if [[ "${DARWIN_SDK_DEPLOYMENT_TARGETS}" != "" ]]; then
-                local IFS=";"; DARWIN_SDK_DEPLOYMENT_TARGETS=($DARWIN_SDK_DEPLOYMENT_TARGETS)
+                local IFS=";"; DARWIN_SDK_DEPLOYMENT_TARGETS=($DARWIN_SDK_DEPLOYMENT_TARGETS); unset IFS
 
                 for target in "${DARWIN_SDK_DEPLOYMENT_TARGETS[@]}"; do
-                    local IFS="-"; triple=($target)
+                    local IFS="-"; triple=($target); unset IFS
```

Comment:
Can you add a comment explaining why this is important?

Reply:
Rationale explanation added as part of c1bdb4f221feb741374715f601e6ea12bb24491e

Reply:
I wonder if we can get away with some regular expressions instead of using ifs and trying to parse out the triple.  We really want to discourage people doing much in this script.  But, if thats not easy, this is okay to merge.  It is correcting a defect, which existed previously.